src/service/stamp/index.ts

```diff
@@ -24,19 +24,14 @@ export async function pickStamp(beeDebug: BeeDebug, console: CommandLog): Promis
     exit(1)
   }
 
-  const filterChoices = stamps.filter(stamp => stamp.batchTTL > 0)
-  const choices = filterChoices.map(
-    stamp => `${stamp.batchID} (${stamp.usageText}) expires in ${secondsToDhms(stamp.batchTTL, true)}`,
-  )
+  const choices = stamps
+    .filter(stamp => stamp.batchTTL > 0)
+    .map(stamp => `${stamp.batchID} (${stamp.usageText}) expires in ${secondsToDhms(stamp.batchTTL, true)}`)
```

Comment:
We have this check:

```
if (!stamps.length) {
    console.error('You need to have at least one stamp for this action.')
    exit(1)
}
```

It should come after this part, and with `stamps` replaced with `choices`. Because right now it is possible that the user only has expired stamps and they are prompted with an empty list.